edison/static/js/policy.controller.js

```diff
@@ -125,16 +124,15 @@ function showCondition(policy, countCondition, elementID) {
     element.appendChild(elementCurrCondition);
 }
 
-function initSettingToNewPolicy() {
-    policy.reset();
-
-    for (var i = 0; i < countCondition; i++) {
+function initSettingToNewPolicy() {  
+    var len = policy.getCountCondition();
+    
+    for (var i = 0; i < len; i++) {
         var element = document.getElementById(`option${i}`);
-
         element.remove();
```

Comment:
You should check if the element is found before you attempt to remove it.

```
if(element) {
  element.remove();
}
```

Reply:
but I added it in the elements function showCondition

Reply:
I see. There's a lot of steps going on beforehand so it's always good to be safe.
But I looked at showCondition and it seems fine, so probably it's not necessary here :)